src/components/HelpResources/HelpResources.js

```diff
@@ -1,12 +1,134 @@
+                  </RouterLink>
+                </CardFooter>
+              </Card>
+
+              <Card gridLayout={{ tablet: { col: 4 } }}>
+                <CardHeader align="center">
+                  <h2>Release Notes</h2>
+                  <IconFlag
+                    size={4}
+                    title="Icon for other card"
+                    className="palette-color-grayscale"
+                  />
+                </CardHeader>
+
+                <CardBody>
+                  <p>Information on CAM API Updates.</p>
+                </CardBody>
+
+                <CardFooter>
+                  <RouterLink to="/release-notes">Release Notes</RouterLink>
+                </CardFooter>
+              </Card>
+
+              <Card gridLayout={{ tablet: { col: 4 } }}>
+                <CardHeader align="center">
+                  <h2>FAQs</h2>
+                  <IconHelpOutline
+                    size={4}
+                    title="Icon for other card"
+                    className="palette-color-grayscale"
+                  />
+                </CardHeader>
+
+                <CardBody>
+                  <p>Frequently Asked Questions about the CAM API.</p>
+                </CardBody>
+
+                <CardFooter>
+                  <RouterLink to="/faq">FAQ</RouterLink>
+                </CardFooter>
+              </Card>
+
+              {/* <Card gridLayout={{ tablet: { col: 4 } }}>
+                <CardHeader align="center">
+                  <h2>API Guides</h2>
+                  <IconMap
+                    size={4}
+                    title="Icon for other card"
+                    className="palette-color-grayscale"
+                  />
+                </CardHeader>
+
+                <CardBody>
+                  <p>View guides on how to use CAMD's API.</p>
+                </CardBody>
+
+                <CardFooter>
+                  <RouterLink to="/api-guides">API Guides</RouterLink>
+                </CardFooter>
+              </Card> */}
+
+              <Card gridLayout={{ tablet: { col: 4 } }}>
+                <CardHeader align="center">
+                  <h2>Contact Us</h2>
+                  <IconAlternateEmail
+                    size={4}
+                    title="Icon for other card"
+                    className="palette-color-grayscale"
+                  />
+                </CardHeader>
+
+                <CardBody>
+                  <p>Reach out to CAMD for further questions.</p>
+                </CardBody>
+
+                <CardFooter>
+                  <a
+                    href="https://www.epa.gov/airmarkets/forms/cam-api-contact-us"
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    Contact Us
+                  </a>
+                </CardFooter>
+              </Card>
+            </CardGroup>
+          </article>
+        </Grid>
+      </Grid>
+    </GridContainer>
```

Comment:
@sgillEPA - remove align="center" from CardHeader tag, add it into Card tag will center all Card contents alignment.